### PR TITLE
Store ad rotation status on offer

### PR DESCRIPTION
**Pull request by davidfischer**
This will allow us to check how rotated ads are performing relative to others. It will also let us check view rates on rotated ads.

### CODE REVIEW

File: adserver/api/views.py [hunk 1]
```diff
@@ -157,7 +157,15 @@ class AdDecisionView(GeoIpMixin, APIView):
...
-        self, ad, placement, publisher, keywords, url, forced=False, paid_eligible=False
+        self,
+        ad,
+        placement,
+        publisher,
+        keywords,
+        url,
+        forced=False,
+        paid_eligible=False,
+        rotated=False,
```
Comment (davidfischer) [inline adserver/api/views.py:168]: Thoughts on using `kwargs` for the default args?

Comment (ericholscher) [reply adserver/api/views.py:168]: Seems fine.. just to make the function sig smaller?

File: adserver/models.py [hunk 2]
```diff
@@ -2425,6 +2431,10 @@ class AdBase(TimeStampedModel, IndestructibleModel):
         null=True,
     )
 
+    # Whether this ad was rotated by the client.
+    # This is needed to measure CTR of rotated ads independently
+    rotated = models.BooleanField(_("Ad was rotated"), default=None, null=True)
```
Comment (ericholscher) [inline adserver/models.py:2436]: Do we want an int here to know how many rotations? Not sure we care, but might be interesting to compare at some point.

**Review by ericholscher — APPROVED** [review]

File: adserver/models.py (hunk 2 above)
Comment (davidfischer) [reply adserver/models.py:2436]: Easy enough to switch

**Pull request merged**